packages/svelte/src/lib/ChoroplethChart.svelte

```diff
@@ -1,6 +1,6 @@
 <script lang="ts">
 	import {
-		ExperimentalChoroplethChart as ChoroplethChartCore,
+		ChoroplethChart as ChoroplethChartCore,
```

Comment:
we should still export both and mark as deprecated

Reply:
The export in packages/svelte/src/lib/index.ts exports both and marks ExperimentalChoroplethChart as deprecated.